frontend/src/AlertsManagement/AlertsList.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import Checkbox from '@material-ui/core/Checkbox';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Tooltip from '@material-ui/core/Tooltip';
+
+class AlertsList extends Component {
+  render() {
+    const alerts = this.props.alerts;
+
+    return (
+      <List className="alerts-list">
+        {/* alert represents the index of the actual alert in allAlerts array. */}
+        {alerts.map((alert, value) => {
```

Comment:
same here with alert variable naming, from my understanding this is some alert number?

Reply:
see explanation above about how the arrays are hardcoded right now. I changed the name to alertIndex to hopefully make it more clear